packages/ts/src/components/donut/config.ts

```diff
@@ -42,6 +42,12 @@ export interface DonutConfigInterface<Datum> extends ComponentConfigInterface {
   showBackground?: boolean;
   /** Background angle range. When undefined, the value will be taken from `angleRange`. Default: `undefined` */
   backgroundAngleRange?: [number, number];
+
+  /** Vertical offset of the half-donut label. */
+  halfDonutLabelOffsetY?: number;
```

Comment:
Generally the component knows very little about what a half donut is, so I think it's going to be confusing to the users too. I think we should generalize this and make these properties work with the regular donut too. Then the names can be  `centralLabelsOffsetX` and `centralLabelsOffsetY`.

Reply:
It's specific to the half-donut though, since they are conditionally applied only in half donut cases.

If it's not a half-donut, these options are not used.

Reply:
We could also introduce `centralLabelsOffsetX` and `centralLabelsOffsetY` for all cases.

Reply:
> We could also introduce centralLabelsOffsetX and centralLabelsOffsetY for all cases.

Yes, that's what I meant. It can be a useful feature for arbitrary angles for example

Reply:
It's a great idea! Here's a separate PR that just adds those two parameters:

https://github.com/ExaForce/unovis/pull/9

If we're ultimately going in the direction of measuring the text and calculating offsets for perfect alignment, that can be treated separately. Maybe that part could be behind another option like `enableHalfDonutAutomaticLabelPositioning`. When that is `true`, then the `centralLabelsOffsetX` and `centralLabelsOffsetY` values could offset the labels relative to where they would automatically be positioned. That would be useful if the design calls for something other than perfect alignment.